snuba/migrations/snuba_migrations/profiles/0001_profiles.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, NamedTuple, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
+    # primary key
+    Column("organization_id", UInt(64)),
```

Comment:
Why do we need the organization id? Do we expect queries across all the projects of an organization ? If not you don't need it here.

Reply:
I guess it's not really necessary. Maybe in case we need to delete all data for a given organization, otherwise I believe we'll always have a project ID to select by.

We discussed that earlier and there didn't seem to be a strong opinion to remove it so I kept it https://github.com/getsentry/snuba/pull/2396#discussion_r806061404 .